### PR TITLE
Update Deprecated Method Usage

### PR DESCRIPTION
**Pull request by harrysolovay**
Part of addressing MinaFoundation/mina-fungible-token#75

### CODE REVIEW

File: src/lib/mina/token/token-contract.ts [hunk 1]
```diff
@@ -162,16 +162,16 @@ abstract class TokenContract extends SmartContract {
...
       from.label = `${this.constructor.name}.transfer() (from)`;
     }
     if (to instanceof PublicKey) {
-      to = AccountUpdate.defaultAccountUpdate(to, tokenId);
+      to = AccountUpdate.default(to, tokenId);
       to.label = `${this.constructor.name}.transfer() (to)`;
     }
 
-    from.balanceChange = Int64.from(amount).neg();
+    from.balanceChange = Int64.from(amount).negV2();
```
Comment (mitschabaude) [inline src/lib/mina/token/token-contract.ts:174]: this is a breaking change, so this PR needs to target the `v2` branch!

**Review by mitschabaude — APPROVED** [review]

**Comment by mitschabaude** [comment]
to pass CI, still needs
```
npm run dump-vks
```
to update verification key regression test

**Comment by harrysolovay** [comment]
Gotcha. Gonna hold off on making any changes until the dust settles re. #1742

**Comment by Trivo25** [comment]
`neg()` has already been moved into `negV2()`, but let's merge it nonetheless for the use of `AccountUpdate.default()` :)

**Pull request merged**